README.md

```diff
@@ -7,11 +7,11 @@ An RDF library in Scala
 -----------------------
```

Comment:
the links can be changed to 
```
[![Build Status](https://secure.travis-ci.org/banana-rdf/banana-rdf.png)](http://travis-ci.org/banana-rdf/banana-rdf)
[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/banana-rdf/banana-rdf?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
[![Maven Central](https://img.shields.io/maven-central/v/org.w3/banana-rdf_2.11.svg)](https://maven-badges.herokuapp.com/maven-central/org.w3/banana-rdf_2.11)
```

This should also fix https://github.com/banana-rdf/banana-rdf/issues/253

Also, some other links on the page are broken, too.